c/tskit/tables.h

```diff
@@ -636,7 +636,9 @@ typedef struct {
 #define TSK_CHECK_SITE_ORDERING (1 << 2)
 #define TSK_CHECK_SITE_DUPLICATES (1 << 3)
 #define TSK_CHECK_MUTATION_ORDERING (1 << 4)
-#define TSK_CHECK_INDEXES (1 << 5)
```

Comment:
Might be easier to follow if we kept the NO_CHECKs grouped together. Also no real reason to change the value of CHECK_INDEXES here - we should leave it as (1<<5).